scripts/shared/lib/deploy_operator

```diff
@@ -53,7 +55,18 @@ function subctl_install_subm() {
 }
 
 function install_subm_all_clusters() {
-    run_parallel "{1..3}" subctl_install_subm
+    if [[ $globalnet = "true" ]]; then
+        # When Globalnet is enabled, each cluster gets a non-overlapping globalCIDR from
```

Comment:
If you *really* want to, can optimize this by starting parallel run on second once first one's global cluster is allocated., so on and so forth. Something to consider if we find deployment times to be too high.

Reply:
That would be rather hard to find out since the allocation of global CIDR is part of the subctl workflow.
What would be possible is to either fix subctl (or whatever does the allocation) to handle parallel joins (which makes sense anyhow), or add a pair of commands `allocate/release-global-cidr` which could be run serially and a flag to set the global cidr on join.